src/devices/ExplorerHat/Lighting/Led.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Device.Gpio;
+using ExplorerHat.Gpio;
+
+namespace Iot.Device.ExplorerHat.Lighting
+{
+    /// <summary>
+    /// Represents a led light
+    /// </summary>
+    public class Led : IDisposable
```

Comment:
would it make sense instead of this class have properties in the parent class with something like:

`Color LedColor` or `bool Led1On` etc

Reply:
Same as Lights.cs